### PR TITLE
Add count GraphQL guide

### PR DESCRIPTION
**Pull request by lauriejim**
#### Description of what you did:

This PR adds a Guide to the docs.
Some people from the community asked about `count` function in GraphQL.
Aggregation is not working well and is not available for every database, this guide let you understand how to add custom GraphQL query, in this case, a `count` query.

### CODE REVIEW

File: docs/3.0.0-beta.x/guides/count-graphql.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+With this guide we will code our own count query.
+
+Here is the [GraphQL documentation](../plugins/graphql.md#customise-the-graphql-schema) we will use to achieve our goal.
+
+## Setup the application
+
+In this example, we will use a **Restaurant** API.
+
+Make sure your have a **Content Type** with some entries.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/count-graphql.md]: make sure you have

File: docs/3.0.0-beta.x/guides/count-graphql.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Create schema.grahql file
+
+To be able to add a new custom query (or mutation), we will have to create a `schema.grahql` file in your **Restaurant** API.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/count-graphql.md]: We can create a `schema.graphql.js`  this way editors wont mistake it with a realy grahpql file.

**Comment by derrickmehaffy** [comment]
Would this not just be easier to add to the default schema until aggregations can be reviewed?

**Comment by alexandrebodin** [comment]
@derrickmehaffy that basically means starting implementing aggregations in bookshelf :D 
We should start working on it "soon"

**Comment by derrickmehaffy** [comment]
> @derrickmehaffy that basically means starting implementing aggregations in bookshelf :D
> We should start working on it "soon"

I mean in this case just handling count since it's just a basic one that's calling the service but I do see your point. (In my case I just handle this in a header on all requests so I don't have to manually call counts)

**Comment by derrickmehaffy** [comment]
@lauriejim fixed the PR feedback for you with a few other changes.

**Comment by alexandrebodin** [comment]
> > @derrickmehaffy that basically means starting implementing aggregations in bookshelf :D
> > We should start working on it "soon"
> 
> I mean in this case just handling count since it's just a basic one that's calling the service but I do see your point. (In my case I just handle this in a header on all requests so I don't have to manually call counts)

Yes we really need to introduce real pagination :)

**Comment by derrickmehaffy** [comment]
> > > @derrickmehaffy that basically means starting implementing aggregations in bookshelf :D
> > > We should start working on it "soon"
> > 
> > 
> > I mean in this case just handling count since it's just a basic one that's calling the service but I do see your point. (In my case I just handle this in a header on all requests so I don't have to manually call counts)
> 
> Yes we really need to introduce real pagination :)

Yup still got that discussion issue open about that and other parameters :wink:

**Pull request merged**